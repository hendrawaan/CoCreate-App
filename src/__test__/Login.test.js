import "@testing-library/jest-dom/extend-expect";
import React from "react";
import { create } from "react-test-renderer";

describe("Test halaman login", () => {
  test("Harus sesuai dengan snapshot komponen heading", () => {
    const Login = create(<Login />);
    expect(Login.toJSON()).toMatchSnapshot();
  });
});
