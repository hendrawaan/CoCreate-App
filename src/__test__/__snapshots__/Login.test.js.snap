// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Halaman Login Halaman Login Harus sesuai dengan snapshot komponen login 1`] = `
<div
  className="container"
>
  <div
    className="align-items-center row-login vh-fit row"
  >
    <div
      className="d-none d-lg-block col-lg-9"
    >
      <img
        alt="Login Illustration"
        className="w-100 login-illustration"
        src="login-illustration.svg"
      />
    </div>
    <div
      className="my-4 col-lg-3"
    >
      <h1
        className="mb-4 text-center"
      >
        Login
      </h1>
      <form
        className=""
        onSubmit={[Function]}
      >
        <div
          className="form-group"
        >
          <input
            autoComplete="email"
            className="form-control"
            id="email"
            name="email"
            onChange={[Function]}
            placeholder="Masukkan email"
            required={true}
            type="email"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            autoComplete="current-password"
            className="form-control"
            id="password"
            name="password"
            onChange={[Function]}
            placeholder="Masukkan password"
            required={true}
            type="password"
          />
        </div>
        <small>
          Lupa password? 
          <a
            href="./"
          >
            Klik disini
          </a>
           
          <br />
          <span
            className="text-danger "
          />
        </small>
        <button
          className="my-3 btn btn-outline-secondary btn-block"
          disabled={false}
          type="submit"
        >
          <svg
            className="mr-2"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
            />
          </svg>
          <span>
            Masuk dengan Email
          </span>
        </button>
      </form>
      <button
        className="my-3 btn btn-outline-dark btn-block"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        <svg
          className="mr-2"
          enableBackground="new 0 0 48 48"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          version="1.1"
          viewBox="0 0 48 48"
          width="1em"
          x="0px"
          xmlns="http://www.w3.org/2000/svg"
          y="0px"
        >
          <path
            d="M43.611,20.083H42V20H24v8h11.303c-1.649,4.657-6.08,8-11.303,8c-6.627,0-12-5.373-12-12
	c0-6.627,5.373-12,12-12c3.059,0,5.842,1.154,7.961,3.039l5.657-5.657C34.046,6.053,29.268,4,24,4C12.955,4,4,12.955,4,24
	c0,11.045,8.955,20,20,20c11.045,0,20-8.955,20-20C44,22.659,43.862,21.35,43.611,20.083z"
            fill="#FFC107"
          />
          <path
            d="M6.306,14.691l6.571,4.819C14.655,15.108,18.961,12,24,12c3.059,0,5.842,1.154,7.961,3.039l5.657-5.657
	C34.046,6.053,29.268,4,24,4C16.318,4,9.656,8.337,6.306,14.691z"
            fill="#FF3D00"
          />
          <path
            d="M24,44c5.166,0,9.86-1.977,13.409-5.192l-6.19-5.238C29.211,35.091,26.715,36,24,36
	c-5.202,0-9.619-3.317-11.283-7.946l-6.522,5.025C9.505,39.556,16.227,44,24,44z"
            fill="#4CAF50"
          />
          <path
            d="M43.611,20.083H42V20H24v8h11.303c-0.792,2.237-2.231,4.166-4.087,5.571
	c0.001-0.001,0.002-0.001,0.003-0.002l6.19,5.238C36.971,39.205,44,34,44,24C44,22.659,43.862,21.35,43.611,20.083z"
            fill="#1976D2"
          />
        </svg>
        Masuk dengan Google
      </button>
      <small>
        Belum punya akun? 
        <span
          className="text-primary"
          onClick={[Function]}
          type="button"
        >
          Klik disini
        </span>
        <br />
      </small>
    </div>
  </div>
</div>
`;
